app/src/main/java/com/github/swent/echo/data/model/Tag.kt

```diff
@@ -1,3 +1,10 @@
 package com.github.swent.echo.data.model
 
-data class Tag(val tagId: String, val name: String)
+/**
+ * Represents a tag.
+ *
+ * @param tagId The ID of the tag.
+ * @param name The name of the tag.
+ * @param isSelected Whether the tag is selected.
+ */
+data class Tag(val tagId: String, val name: String, val isSelected: Boolean)
```

Comment:
I'm not sure if having `isSelected` on the data model makes sense. This is state only concerning the UI and nothing we want to store. See also my comment here: https://github.com/echo-swent/echo/pull/31#issuecomment-2047914993